game-core/src/test/java/games/strategy/engine/data/annotations/ExampleAttachment.java

```diff
@@ -84,13 +94,13 @@ public String getNotAProperty() {
    * Adds to, not sets. Anything that adds to instead of setting needs a clear function as well.
    */
   @GameProperty(xmlProperty = true, gameProperty = true, adds = true)
-  public void setGivesMovement(final String value) {
+  public IntegerMap<UnitType> createGivesMovement(final String value) {
```

Comment:
Some of the tests in `ValidateAttachmentsTest` are failing because

> Class games.strategy.engine.data.annotations.ExampleAttachment setter createGivesMovement: does not begin with 'set' but has GameProperty annotation